python/modules/IcePy/Init.cpp

```diff
@@ -130,14 +130,15 @@ PyMODINIT_FUNC
 #else
 PyMODINIT_FUNC ICE_DECLSPEC_EXPORT
 #endif
-
 PyInit_IcePy(void)
 {
     PyObject* module;
 
     Ice::registerIceSSL(false);
     Ice::registerIceDiscovery(false);
     Ice::registerIceLocatorDiscovery(false);
+    Ice::registerIceUDP(true);
```

Comment:
I don't think UDP is strictly necessary as IceDiscovery registers it. But probably best to have it.

```
#ifdef ICE_STATIC_LIBS
        //
        // Also register the UDP plugin with static builds to ensure the UDP transport is loaded.
        //
        registerIceUDP(true);
#endif
```